src/megameklab/com/util/MenuBarCreator.java

```diff
@@ -1390,4 +1289,8 @@ public void lostOwnership(Clipboard arg0, Transferable arg1) {
 
     }
 
+    private final FilenameFilter unitFilesFilter =
```

Comment:
Do these endsWith need to be case sensitive?

Reply:
Good catch. I'll need to fix that.